mail-filter/tests/data/github-merged-email.txt

```diff
@@ -5,7 +5,7 @@ Date: Fri, 04 Nov 2022 06:18:29 -0700
 From: Some User <notifications@github.com>
 Reply-To: UbuntuBudgie/budgie-desktop <reply+AKT4UUMF3LHYRIQTBKZCJRGBOJB2LEVBNHHFICBOZM@reply.github.com>
 To: UbuntuBudgie/budgie-desktop <budgie-desktop@noreply.github.com>
-Cc: Janitor Bot <janitor@jelmer.uk>,
+Cc: Janitor <janitor@jelmer.uk>,
```

Comment:
Please don't change. This is test data, and came from an actual email received from GitHub.

Reply:
Reverted